addons/udes_stock/tools/datetime.py

```diff
@@ -0,0 +1,20 @@
+from dateutil.parser import parse
+import dateutil.tz
+from datetime import datetime
+
+
+def format_iso_datetime(dt):
+    """
+    Converts date strings returned by Odoo, or a datetime object, to 
+    ISO 8601 date strings with the timezone included.
+
+    eg. '2020-01-07 18:01:01' gets turned into '2020-01-07T18:01:01+00:00'
+    """
+    if not isinstance(dt, datetime):
+        dt = parse(dt)
+    dt = dt.replace(tzinfo=dateutil.tz.tzutc())
+    # The timespec option is only available in python >=  3.6
```

Comment:
We can probably drop this comment now, we no longer support 3.5

Reply:
Is a copy paste from the one in web module :), Happy to remove the comment